docs/release-notes/experiment-tag-propagation.rst

```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+**New Feature**
+
+-  Cluster: Add support for experiment tag propagation.
+
+   -  The enterprise edition of [HPE Machine Learning Development Environment]
+      (https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html)
+      now allows for experiment tags to be propagated as labels to the associated jobs on the HPC
```

Comment:
Might be helpful to specifically mention the addition of ``resource_manager.job_project_source`` here so that they can actually find the feature.

Reply:
Done.